lib/dkim.js

```diff
@@ -463,8 +464,9 @@ function DKIMVerify(email, callback, sigs) {
 
         // The other DNS errors are operational in nature;
         sigs.result = sig.result = false;
-        sigs.issue_desc = sig.issue_desc = err;
-        return callback(err, sigs);
+        sigs.issue_desc = sig.issue_desc = err.message;
+
+        return callback(null, sigs); // returning the error AND the sigs breaks promisification... should do callback(null, sigs)
```

Comment:
returning both shouldn't have any effect on promisification? although i think maybe the issue is that the error is returned at all (ie i think `callback(err)` would break the same way as it does currently) ... so we intend to swallow all these errors is that correct?